### PR TITLE
Edit of Health Checks page

### PR DESCRIPTION
**Pull request by sascala**

### CODE REVIEW

File: docs/docs/health-checks.md [hunk 1]
```diff
@@ -4,76 +4,37 @@ title: Health Checks
...
 # Health Checks
 
-Health checks may be specified per application to be run against that application's tasks.
+You can specify health checks against the tasks of individual applications.
 
-- The default health check defers to Mesos' knowledge of the task state `TASK_RUNNING => healthy`
-- Marathon provides a `health` member of the task resource
-  via the [REST API]({{ site.baseurl }}/docs/rest-api.html).
+- The default health check employs Mesos' knowledge of the task state `TASK_RUNNING => healthy`
+- Marathon provides a `health` member of the task resource via the [REST API]({{ site.baseurl }}/docs/rest-api.html).
```
Comment (sascala) [inline docs/docs/health-checks.md]: What does this mean?

File: docs/docs/health-checks.md [hunk 2]
```diff
@@ -83,22 +44,22 @@ Options applicable to every protocol:
   health checks.
 * `maxConsecutiveFailures`(Optional. Default: 3): Number of consecutive health
   check failures after which the unhealthy task should be killed. If this value
-  is `0`, then tasks will not be killed due to failing this check.
+  is `0`, tasks will not be killed if they fail the health check.
 * `timeoutSeconds` (Optional. Default: 20): Number of seconds after which a
   health check is considered a failure regardless of the response.
```
Comment (sascala) [inline docs/docs/health-checks.md]: What do we mean by "regardless of response"?

File: docs/docs/health-checks.md (hunk 1 above)
Comment (aquamatthias) [inline docs/docs/health-checks.md]: There is a dedicated health check definition inside an application definition.

File: docs/docs/health-checks.md (hunk 2 above)
Comment (aquamatthias) [inline docs/docs/health-checks.md]: A response after the timeout (even a healthy one) is considered unhealthy.

**Comment by aquamatthias** [comment]
Close for now - please reopen when ready.

**Pull request closed without merging**